testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ee/concurrent/HungTasksTerminationTestCase.java

```diff
@@ -0,0 +1,301 @@
+        final String jndiName = "java:jboss/ee/concurrency/executor/" + RESOURCE_NAME;
+        addOperation.get(ManagedExecutorServiceResourceDefinition.JNDI_NAME).set(jndiName);
+        addOperation.get(ManagedExecutorServiceResourceDefinition.HUNG_TASK_THRESHOLD).set(HUNG_TASK_THRESHOLD_TEST_VALUE);
+        final ModelNode addResult = managementClient.getControllerClient().execute(addOperation);
+        Assert.assertFalse(addResult.get(FAILURE_DESCRIPTION).toString(), addResult.get(FAILURE_DESCRIPTION).isDefined());
+        try {
+            final ManagedExecutorService executorService = InitialContext.doLookup(jndiName);
+            Assert.assertNotNull(executorService);
+            testHungTasksCancellationOperation(pathAddress, executorService);
+        } finally {
+            // remove
+            final ModelNode removeOperation = Util.createRemoveOperation(pathAddress);
+            removeOperation.get(OPERATION_HEADERS, ALLOW_RESOURCE_SERVICE_RESTART).set(true);
+            final ModelNode removeResult = managementClient.getControllerClient().execute(removeOperation);
+            Assert.assertFalse(removeResult.get(FAILURE_DESCRIPTION).toString(), removeResult.get(FAILURE_DESCRIPTION)
+                    .isDefined());
+        }
+    }
+
+    @Test
+    public void testManagedScheduledExecutorServiceHungTasksCancellationOperation() throws Exception {
+        final PathAddress pathAddress = EE_SUBSYSTEM_PATH_ADDRESS.append(EESubsystemModel.MANAGED_SCHEDULED_EXECUTOR_SERVICE, RESOURCE_NAME);
+        // add
+        final ModelNode addOperation = Util.createAddOperation(pathAddress);
+        final String jndiName = "java:jboss/ee/concurrency/scheduledexecutor/" + RESOURCE_NAME;
+        addOperation.get(ManagedScheduledExecutorServiceResourceDefinition.JNDI_NAME).set(jndiName);
+        addOperation.get(ManagedScheduledExecutorServiceResourceDefinition.HUNG_TASK_THRESHOLD).set(HUNG_TASK_THRESHOLD_TEST_VALUE);
+        final ModelNode addResult = managementClient.getControllerClient().execute(addOperation);
+        Assert.assertFalse(addResult.get(FAILURE_DESCRIPTION).toString(), addResult.get(FAILURE_DESCRIPTION).isDefined());
+        try {
+            // lookup the executor
+            final ManagedExecutorService executorService = InitialContext.doLookup(jndiName);
+            Assert.assertNotNull(executorService);
+            testHungTasksCancellationOperation(pathAddress, executorService);
+        } finally {
+            // remove
+            final ModelNode removeOperation = Util.createRemoveOperation(pathAddress);
+            removeOperation.get(OPERATION_HEADERS, ALLOW_RESOURCE_SERVICE_RESTART).set(true);
+            final ModelNode removeResult = managementClient.getControllerClient().execute(removeOperation);
+            Assert.assertFalse(removeResult.get(FAILURE_DESCRIPTION).toString(), removeResult.get(FAILURE_DESCRIPTION)
+                    .isDefined());
+        }
+    }
+
```

Comment:
@emmartins what about a short description:

/**
     * Tests the `hung-task-termination-period` works on a `managed-executor-service`
     * @throws Exception
     */

Reply:
Used a different wording, to not depend on attributes/operation names, but done.